docs/introduction/developer-quickstart.md

```diff
@@ -91,22 +90,17 @@ contract StorageExample {
 
 ### Explanation:
 
-Since `getValue` is the only function which exposes the values, we can add a check like this: `require(tx.origin == account, "Not authorized!");` 
-If anyone, other than the original account, asks for the value, they will get an error.
+Since `getValue` is the only function which exposes the values, we can add a check like this: `require(tx.origin == account, "Not authorized!");` If anyone, other than the original account, asks for the value, they will get an error.
 
 :::info
 In Ethereum, since all data is accessible anyway, there is no need to sign calls to view functions, so `tx.origin` can be spoofed.
 :::
 
-In Obscuro, the platform ensures that calls to view functions are authenticated. Which means that behind the scenes, 
-there is a signature of the ``tx.origin`` address. 
-
+In Obscuro, the platform ensures that calls to view functions are authenticated. Which means that behind the scenes, there is a signature of the `tx.origin` address.
 
 ## Step 4: Emitting Events
 
-Events in Ethereum are crucial for UIs to react to smart contract state changes.
-In this step we'll emit an event when a user stores a value, and also, to show how popular our contract is,
-we'll emit an event when certain milestones are reached.
+Events in Ethereum are crucial for UIs to react to smart contract state changes. In this step, we'll emit an event when a user stores a value, and also, to show how popular our contract is, we'll emit an event when certain milestones are reached.
```

Comment:
Events in Ethereum are crucial for UIs to react to smart contract state changes. In this step, we'll emit an event when a user stores a value. We'll also gauge the popularity of our contract by emitting an event when certain milestones are reached.

Reply:
Fixed